db/app/controllers/articles.controller.js

```diff
@@ -33,10 +34,53 @@ exports.getArticleComments = (req, res, next) => {
   ];
   Promise.all(promises)
     .then((promiseResolutions) => {
-      console.log(promiseResolutions);
       res.status(200).send({ comments: promiseResolutions[0] });
     })
     .catch((err) => {
       next(err);
     });
 };
+
+exports.postComment = (req, res, next) => {
+  const { article_id } = req.params;
+  const comment = req.body;
+  fetchArticleById(article_id)
```

Comment:
article_id is also a foreign key so in this case a separate check that the article exists should not be necessary as a psql error will be thrown anyway